x/concentrated-liquidity/types/keys.go

```diff
@@ -62,14 +66,17 @@ func KeyTickPrefix(poolId uint64) []byte {
 	return key
 }
 
+// KeyFullPosition uses pool Id, owner, lower tick, upper tick, and frozenUntil for keys
+func KeyFullPosition(poolId uint64, addr sdk.AccAddress, lowerTick, upperTick int64, frozenUntil time.Time) []byte {
+	frozenUntilKey := osmoutils.FormatTimeString(frozenUntil)
+	addrKey := address.MustLengthPrefix(addr.Bytes())
+	return []byte(fmt.Sprintf("%s%s%s%s%d%s%d%s%d%s%s", PositionPrefix, KeySeparator, addrKey, KeySeparator, poolId, KeySeparator, lowerTick, KeySeparator, upperTick, KeySeparator, frozenUntilKey))
```

Comment:
Is there future needs to have these values separated and parsed?

Reply:
Does having the separator make some kind of meaningful overhead? It seems better to have it then to not have it, but I could be wrong (at least that was the conclusion I came to when I brought it up with Roman)